src/Tools/dotnet-dump/Commands/SetClrPathCommand.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.DebugServices;
+using Microsoft.Diagnostics.Repl;
+using System.CommandLine;
+
+namespace Microsoft.Diagnostics.Tools.Dump
+{
+    [Command(Name = "setclrpath", Platform = CommandPlatform.Linux | CommandPlatform.OSX, Help = "Set the path to load coreclr DAC/DBI files.")]
```

Comment:
Any reason this is limited to not windows? Seems like it could be useful for swapping from FX to Core

Reply:
SOS on Windows doesn't use the host function in LLDBServices.GetCoreClrDirectory.  When SOS on xplat under lldb, the lldbplugin (which looks like the "host") implements this command and that LLDBServices interface.

I do want a way to specific the runtime directory path and which runtime (clr or coreclr) to use on Windows, but it require some more thought and reworking of some of the SOS logic.  It currently for Windows calls various dbgeng interface functions to find the runtime module. See https://github.com/dotnet/diagnostics/blob/49ff7a20915ed09002b795972d47cb18423830d0/src/SOS/Strike/hostcoreclr.cpp#L248